packages/client/src/comp/GetInvolved.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import bgImage from './img/get_involved_bg.jpeg'
+import iconBadge from './img/gi_icon_badge.svg'
+import iconGear from './img/gi_icon_gear.svg'
+import { useAppHistory } from '../lib/path'
+import { cssQuery } from './util/cssQuery'
+import { MarketingWrapper } from './util/MarketingWrapper'
+import { Container } from 'muicss/react'
+import { UnderlineLink } from './util/UnderlineLink'
+
+const bgGradient = 'linear-gradient(to left, rgba(0, 89, 186, 0.9), rgba(0, 89, 186, 0.9))'
+
+const Wrapper = styled(MarketingWrapper)`
+  background: ${bgGradient}, url(${bgImage});
+  ${cssQuery.desktop.all} { padding: 150px 0; }
+  background-size: cover;
+  .mui-container {
+    justify-content: center;
+  }
+`
+
+const CardWrapper = styled.div`
+  width: 100%;
+  ${cssQuery.desktop.wide} { width: 80%; }
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex-flow: row wrap;
+  align-items: stretch;
+  justify-content: center;
+  ${cssQuery.desktop.all} {
+    justify-content: space-between;
+  }
+`
+
+const Card = styled.div`
+  background-color: white;
+  width: 90%;
+  ${cssQuery.mobile.wide} { width: 65%; }
+  ${cssQuery.desktop.all} { width: 49%; }
+  margin: 1em 0;
+  padding: 25px;
+  padding-bottom: 50px;
+  box-sizing: border-box;
+
+  img {
+    height: 25%;
+  }
+
+  p {
+    margin: 2em 0;
+    a {
+      text-decoration: underline;
+      color: #323232;
+    }
+  }
+
+  b { color: #db002f; }
+
+  /* Send "Contact US" to the bottom */
+  position: relative;
+  & > a {
```

Comment:
`& > a` is a bit subtle -- why don't we make it `& > .align-bottom` or something like that?